### DISCUSSION TITLE
What is the difference between the two perl installations on Windows?

### DISCUSSION THREAD

**Discussion by hakonhagland** [shogo82148/actions-setup-perl#1842] [General]
I know what Strawberry perl is, but what is the other default perl that is installed with the action?

**Comment by shogo82148** [comment]
The other Perl are built by me.

**Reply by hakonhagland** [reply]
Thanks! That is interesting :smile:  How is that done? For example, are you using a MSYS2 C compiler or Visual Studio C compiler.

**Reply by shogo82148** [reply]
MSYS2 C compiler that Strawberry Perl provides.
I built them on GitHub Actions. The workflow is here: https://github.com/shogo82148/actions-setup-perl/blob/main/.github/workflows/win32.yml
